tests/python/unittest/test_runtime_module_based_interface.py

```diff
@@ -688,6 +688,36 @@ def test_num_threads():
         assert reported == hardware_threads or reported == hardware_threads // 2
 
 
+@tvm.testing.requires_llvm
+def test_graph_module_zero_copy():
```

Comment:
Probably the second test case can be added. This test case should test setting `zero_input` when the keys and values passed through `params`

Reply:
@echuraev a second test case is added for setting params input.